lib/queue_classic_plus/update_metrics.rb

```diff
@@ -18,7 +18,7 @@ def self.metrics
       {
         jobs_queued: jobs_queued,
         jobs_scheduled: jobs_scheduled,
-        max_created_at: max_age("created_at"),
+        max_created_at: max_age("created_at", "created_at = scheduled_at"),
```

Comment:
A yeah, I now understand what this is for. Hum.. It feels like this is a problem that would be better solved in QC itself.

`scheduled_at` should be null when we enqueue now and we should process those jobs instantly. Thoughts?